manifests/RProject/R/4.0.0.yaml

```
@@ -0,0 +1,12 @@
+Id: RProject.R
+Publisher: RProject
+Name: RProject R
+Version: 4.0.0
+License: GNU General Public License, version 2
+LicenseUrl: https://www.r-project.org/Licenses/
+Description: The base binary distribution of R statistics language.
+InstallerType: exe
+Installers:
+  - Arch: x64
+    Url: https://cloud.r-project.org/bin/windows/base/R-4.0.0-win.exe
+    Sha256: CA0C28A4F3E0AE66F64F01024BCCF3F3477C6E1008698AB96CEB3162E24E6C87
```

I would add some switches as the R installer allows it
```suggestion
    Sha256: CA0C28A4F3E0AE66F64F01024BCCF3F3477C6E1008698AB96CEB3162E24E6C87
    Switches:
      Silent: /VERYSILENT
      SilentWithProgress: /SILENT
      InstallLocation: /DIR="<INSTALLPATH>"
```